packages/crypto/src/lib/crypto.ts

```diff
@@ -372,6 +373,103 @@ async function getAmdCert(url: string): Promise<Uint8Array> {
   }
 }
 
+export const walletEncrypt = async(
+  myWalletSecretKey: Uint8Array, 
+  theirWalletPublicKey: Uint8Array,
+  sessionSig: Uint8Array,
+  message: Uint8Array
+): Promise<WalletEncryptedPayload> => {
+  const random = new Uint8Array(16);
+  window.crypto.getRandomValues(random);
```

Comment:
Using `window` won't work on node side, neither deno if we want to put the SDK in LAs. However the method exists on nodes and denos crypto module